sphinxcontrib/katex.py

```diff
@@ -359,6 +359,9 @@ class KaTeXServer:
     # Message length is 32-bit little-endian integer
     LENGTH_STRUCT = struct.Struct("<i")
 
+    # Path to KaTeX javascript file
+    KATEX_PATH = None
+
```

Comment:
Do not use capitals for variables.
```suggestion
    # Path to KaTeX javascript file, or None if the bundled should be used.
    katex_path = None

```

Reply:
I changed all class variables of `KaTeXServer` to use lower case letters.